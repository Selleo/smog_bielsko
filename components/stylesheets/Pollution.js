import StyleSheet from 'react-native-extensible-stylesheet'

export default StyleSheet.create({
  bigComponent: {
    alignItems: 'stretch',
    alignSelf: 'stretch',
    flexWrap: 'wrap',
    justifyContent: 'space-between',
    paddingHorizontal: 10,
    paddingVertical: 15,
  },
  elevatedView: {
    alignItems: 'center',
    backgroundColor: 'white',
    justifyContent: 'space-between',
    margin: 0,
  },
  singleCTitle: {
    marginLeft: 5
  },
  singleElevatedView: {
    marginBottom: 15
  }
});
