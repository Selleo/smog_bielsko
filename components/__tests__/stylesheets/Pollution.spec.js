import 'react-native';
import React                          from 'react';

import { shallow }                    from 'enzyme';
import chai, { expect }               from 'chai';
import dirtyChai                      from 'dirty-chai';

import Pollution                         from '../../stylesheets/Pollution';

const originalExpect = global.expect;

chai.use(dirtyChai);

describe('Pollution', () => {
  it('renders correctly', () => {
    originalExpect(Pollution).toMatchSnapshot();
    expect(Pollution).to.exist();
  });
});
