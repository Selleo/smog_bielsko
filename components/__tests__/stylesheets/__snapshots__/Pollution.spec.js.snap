exports[`Pollution renders correctly 1`] = `
ExtensibleStyleSheet {
  "_options": Object {
    "bigComponent": Object {
      "alignItems": "stretch",
      "alignSelf": "stretch",
      "flexWrap": "wrap",
      "justifyContent": "space-between",
      "paddingHorizontal": 10,
      "paddingVertical": 15,
    },
    "elevatedView": Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "justifyContent": "space-between",
      "margin": 0,
    },
    "singleCTitle": Object {
      "marginLeft": 5,
    },
    "singleElevatedView": Object {
      "marginBottom": 15,
    },
  },
  "_styles": Object {
    "bigComponent": Object {
      "alignItems": "stretch",
      "alignSelf": "stretch",
      "flexWrap": "wrap",
      "justifyContent": "space-between",
      "paddingHorizontal": 10,
      "paddingVertical": 15,
    },
    "elevatedView": Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "justifyContent": "space-between",
      "margin": 0,
    },
    "singleCTitle": Object {
      "marginLeft": 5,
    },
    "singleElevatedView": Object {
      "marginBottom": 15,
    },
  },
  "_variables": Object {},
  "bigComponent": Object {
    "alignItems": "stretch",
    "alignSelf": "stretch",
    "flexWrap": "wrap",
    "justifyContent": "space-between",
    "paddingHorizontal": 10,
    "paddingVertical": 15,
  },
  "elevatedView": Object {
    "alignItems": "center",
    "backgroundColor": "white",
    "justifyContent": "space-between",
    "margin": 0,
  },
  "singleCTitle": Object {
    "marginLeft": 5,
  },
  "singleElevatedView": Object {
    "marginBottom": 15,
  },
}
`;
